### ISSUE TITLE
[FEATURE] Actually fix things when people ask you to.

### ISSUE THREAD

**Issue by qbfdesgfkdsAfjklgskl** [cupiditys/iReady-Overload#259]
I asked you to fix this. "The extension is no longer supplying fresh ass every time I rub the finish lesson button and it is very irritating. I ask you fix the ASAP."

All you did was reply with shitty answers and not say anything about it nor did you fix it.

**Comment by cupiditys** [comment]
You have autism

**Comment by qbfdesgfkdsAfjklgskl** [comment]
No; I believe you do, because you can't understand a single fucking question.

**Comment by qbfdesgfkdsAfjklgskl** [comment]
Bro, please daddy.
